vars/Fedora.yml

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: MIT
+---
+# Put internal variables here with Fedora specific values.
```

Comment:
`__certificate_packages` is missing here. Also probably `__certificate_default_directory: /etc/pki/tls`

Reply:
I think we can remove this file since the vars will be loaded from defaults.

Reply:
Amended.